### PR TITLE
Configurable reverb and chorus for native MIDI

### PR DESCRIPTION
**Pull request by ceski-1**
Addresses #940

### CODE REVIEW

File: src/i_winmusic.c [hunk 1]
```diff
@@ -26,6 +26,18 @@
 #include "m_misc.h"
 #include "midifile.h"
 
+#ifndef BETWEEN
+#define BETWEEN(l,u,x) (((l)>(x))?(l):((x)>(u))?(u):(x))
```
Comment (rfomin) [inline src/i_winmusic.c:30]: `BETWEEN` already defined in `crispy.h`

**Comment by rfomin** [comment]
Thanks for this, but should we add this to Chocolate Doom?

File: src/i_winmusic.c (hunk 1 above)
Comment (ceski-1) [reply src/i_winmusic.c]: Fixed it, thank you.

**Comment by ceski-1** [comment]
> Thanks for this, but should we add this to Chocolate Doom?

I'll let you and @fabiangreffrath decide if the PR should be done with Chocolate and the merged here later.

**Comment by fabiangreffrath** [comment]
@rfomin is right. This isn't Crispy-specific and should go into Choco first.

**Pull request closed without merging**